### PR TITLE
Regenerate ContactType DAO

### PR DESCRIPTION
**Pull request by colemanw**
Overview
----------------------------------------
Regenerates DAO that got out-of-sync after #25069

### CODE REVIEW

File: CRM/Contact/DAO/ContactType.php [hunk 1]
```diff
@@ -308,7 +314,7 @@ public static function &fields() {
         'is_reserved' => [
           'name' => 'is_reserved',
           'type' => CRM_Utils_Type::T_BOOLEAN,
-          'title' => ts('Contact Type is Reserved?'),
```
Comment (eileenmcnaughton) [inline CRM/Contact/DAO/ContactType.php:311]: On the others I changed the `<html><label>` but not the title - cos the title shows in (e.g) views where there are multiple entities

Comment (colemanw) [reply CRM/Contact/DAO/ContactType.php:311]: Ok, but I've taken out the question marks because those are silly

Comment (eileenmcnaughton) [reply CRM/Contact/DAO/ContactType.php:311]: Agree

**Comment by colemanw** [comment]
Unrelated test fails

**Pull request merged**